tests/blocks/block_170.cairo

```diff
@@ -54,8 +54,9 @@ pub fn block_170() -> Block {
                         sequence: 4294967295,
                         witness: array![].span(),
                         previous_output: OutPoint {
-                            txid: 0x0437cd7f8525ceed2324359c2d0ba26006d92d856a9c20fa0241106ee5a597c9_u256
```

Comment:
This was correct, reversed hex coming from RPC is equivalent to little endian encoding of U256